src/components/Tooltip/Tooltip.js

```diff
@@ -87,7 +89,11 @@ export const Tooltip = ({
     eventsEnabled: true,
   }
 
-  const tooltipClasses = [styles.root, inline ? styles.inline : styles.block]
+  const tooltipClasses = [
+    styles.root,
+    inline ? styles.inline : noLayout ? '' : styles.block,
```

Comment:
block has an undesirable margin left and inline has an undesirable white space at the bottom so i just made a new prop so ignore either options. ideally i would delete these styles so tooltips are not forced with a certain [layout](https://seek-oss.github.io/braid-design-system/foundations/layout) but this is done to prevent other uses of tooltips stuff from looking bad.